test/invariants/symbolic/Greeter.t.sol

```diff
@@ -35,7 +35,7 @@ contract SymbolicGreeter is SymTest, Test {
     (_success,) = address(targetContract).call(abi.encodeCall(Greeter.setGreeting, ('')));
 
     // Output condition check
-    vm.assume(_success); // discard failing calls
```

Comment:
is there a reason we exclude instead of testing in the post-condition state? This should rather be `if(success) assert(string != '') else assert(string=='')` no?